schema/metrics/metrics.schema

```diff
@@ -0,0 +1,271 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://opensearch.org/schemas/Metrics",
+  "title": "OpenTelemetry Metrics",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "attributes": {
+      "$ref": "/schemas/Attributes"
+    },
+    "resource": {
+      "type": "object"
+    },
+    "description": {
+      "type": "string"
+    },
+    "unit": {
+      "type": "string"
+    },
+    "kind": {
+      "type": "string",
+      "enum": [
+        "COUNTER_MONOTONIC",
+        "COUNTER_CUMULATIVE",
+        "SUM",
+        "GAUGE",
+        "HISTOGRAM",
+        "EXPONENTIAL_HISTOGRAM"
```

Comment:
this is not fully matching up with readme specs - some of these are more specific, should they be matching?

Reply:
Same comment in a few places

Reply:
I'll update this as well - thanks 🙏